### PR TITLE
ENG-2235: Newsletter signup

### PR DESCRIPTION
**Pull request by deammer**
### What changed:

With https://github.com/Codesee-io/opensourcehub/pull/250, we added a checkbox to OSH that allows users tosubscribe to our newsletter. This boolean value is stored in the `profiles` collection in Firestore as `joinNewsletter`.

We want this field to end up in HubSpot as `send_me_osh_news: true/false`. To do that, we've set up cloud functions that listen to updates in the `profiles` collection. When an item in that collection gets updated, the `joinNewsletter` field is passed to `Segment.identify()` as the `send_me_osh_news` field. Once Segment receives that, it'll send the data into HubSpot according to Eden's setup.

In short:

OSH UI -> Firestore -> Cloud functions -> Segment -> Hubspot

### CODE REVIEW

**Review by joshuadoan — APPROVED** [review]
Thanks for adding comments. Good work.
